ocaml-lsp-server/src/config_data.ml

```diff
@@ -0,0 +1,261 @@
+open Import
+open Import.Json.Conv
+
+module Lens = struct
+  type t = { enable : bool Json.Nullable_option.t [@default Some true] }
```

Comment:
Can we make `enable` to have type of bool, with default value of true?
That way we would have only two possibilities: true or false. Now, we have 3 (Some true, Some false, None), but two of them are equivalent.